src/material/slider/testing/shared.spec.ts

```diff
@@ -1,14 +1,16 @@
 import {HarnessLoader} from '@angular/cdk/testing';
 import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
-import {Component} from '@angular/core';
+import {Component, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 import {MatSliderModule} from '@angular/material/slider';
 import {MatSliderHarness} from '@angular/material/slider/testing/slider-harness';
 
 /** Shared tests to run on both the original and MDC-based sliders. */
 export function runHarnessTests(
     sliderModule: typeof MatSliderModule,
-    sliderHarness: typeof MatSliderHarness) {
+    sliderHarness: typeof MatSliderHarness,
+    supportsVertical: boolean,
+    supportsInvert: boolean) {
```

Comment:
Can we make this an options object instead of flag arguments? I generally want to avoid flag arguments (https://martinfowler.com/bliki/FlagArgument.html).